docs/02-sygma-sdk/05-Examples/01-NodeJS-Token-Transfer-Example.md

```diff
@@ -0,0 +1,345 @@
+  return domain || null;
+}
+```
+
+#### Build transfer object
+
+To be able to use the SDK, we need to build the `transfer` object. For that, we create the following `buildTransfer` function. This function expects several parameters: `wallet`, `from` and `to` destinations, `resource`, and `amount`. All of the parameters are defined in the comment below. Paste in the following snippet into the `transfer.ts` file:
+
+```ts
+// transfer.ts
+/**
+ * Builds a transfer object with the necessary parameters.
+ * @param wallet The Ethereum wallet to use for the transfer.
+ * @param from The domain to transfer from.
+ * @param to The domain to transfer to.
+ * @param resource The resource to transfer.
+ * @param recipient The recipient of the transfer.
+ * @param amount The amount to transfer, in wei.
+ * @returns The transfer object.
+ */
+async function buildTransfer(
+  wallet: Wallet,
+  from: Domain,
+  to: Domain,
+  resource: Resource,
+  recipient: string,
+  amount: string
+): Promise<Transfer<Fungible>> {
+  return {
+    sender: await wallet.getAddress(),
+    amount: {
+      amount,
+    },
+    from,
+    to,
+    resource,
+    recipient,
+  };
+}
+```
+
+#### Create the main `erc20Transfer` function
+
+Now, we create the main `erc20Transfer` function. This function uses all of the functions defined above, along with the Sygma SDK to perform the cross-chain token transfer. Don't forget to type in the constant variables `providerApiKey` and `privateKey` where it prompts you in quotations. 
+
+You should have already obtained `ERC20LRTest` tokens from the faucet.
+
+Paste in the following snippet into the `transfer.ts` file:
+
+```ts
+// transfer.ts
+/**
+ * Transfers ERC20 tokens between two Ethereum networks using the Sygma SDK.
+ * @returns A Promise that resolves when the transfer is complete.
+ */
+export async function erc20Transfer(): Promise<void> {
+  const provider = new providers.JsonRpcProvider(
+    "<PROVIDER_API_KEY>" // type in the const `providerApiKey` without the quotations
+  );
+  const wallet = new Wallet(
+    "<YOUR_PRIVATE_KEY>" as string, // type in the const `privateKey` without the quotations
+    provider
+  );
+
+  const assetTransfer = await initAssetTransfer(provider);
+  const erc20Resource = findERC20Resource(assetTransfer);
+  if (!erc20Resource) {
+    throw new Error("Resource not found");
+  }
+
+  const goerli = findDomainByChainID(assetTransfer, GOERLI_CHAIN_ID);
+  if (!goerli) {
+    throw new Error("Network goerli not supported");
+  }
+
+  const sepolia = findDomainByChainID(assetTransfer, SEPOLIA_CHAIN_ID);
+  if (!sepolia) {
+    throw new Error("Network sepolia not supported");
+  }
+
+  const transfer = await buildTransfer(
+    wallet,
+    goerli,
+    sepolia,
+    erc20Resource,
+    await wallet.getAddress(),
+    "50000000000000000000" // ERC20LRTest token uses 18 decimal places. To transfer 50 tokens, you would need to enter `50` followed by 18 zeroes, or `50000000000000000000`
```

Comment:
here could be better to suggest `ethers.utils.parseEther('5', 18)`

Reply:
updated